webui.py

```diff
@@ -110,18 +111,24 @@ def generate_clicked(task):
                                  elem_id='final_gallery')
             with gr.Row(elem_classes='type_row'):
                 with gr.Column(scale=17):
-                    prompt = gr.Textbox(show_label=False, placeholder="Type prompt here or paste parameters.", elem_id='positive_prompt',
+                    prompt = gr.Textbox(show_label=False, placeholder="Type prompt here or paste parameters.",
+                                        elem_id='positive_prompt',
```

Comment:
Please revert all autoformatting of the code so your PR only has changes of your feature. Code cleanup is done separately.

Reply:
yes, the formatting is done automatically by ide automatically and now it has been fixed 👍